src/components/tabs/Tabs.js

```diff
@@ -0,0 +1,40 @@
+import { TabNav } from './TabNav';
+import { Tab } from './Tab';
+import TaskList from '../taskList';
+
+import './tabs.scss';
+
+export class Tabs extends Component {
+  state = {
+    selectIndex: 0,
+  }
+
+  componentDidMount() {
+    const { dayWeek } = this.props;
+    this.setState({ selectIndex: dayWeek });
+  }
+
+  selectTab = (selectIndex) => {
+    this.setState({ selectIndex });
+  }
+
+  render() {
+    const { days, markDone, delTask } = this.props;
+    const { selectIndex } = this.state;
+
+    return (
+      <div className="tabs">
+        {
+          days && <TabNav tabs={days} selectIndex={selectIndex} selectTab={this.selectTab} />
+        }
+        <div className="tab-content">
+          {
+            days && (
+              <TaskList content={days[selectIndex]} markDone={markDone} delTask={delTask} />
+            )
+          }
+        </div>
```

Comment:
this part should be changed:
```
<div className="tab-content">
    {tabs[selectedIndex].props.children}
</div>
```

With your approach we can't use `Tabs` for other views, only  for days and tasks, but our component is universal - it is just a wrapper for some specific content.